ui/src/Pages/Library/Cards.jsx

```diff
@@ -0,0 +1,174 @@
+import { useCallback, useContext, useEffect, useRef, useState } from "react";
+import { useSelector } from "react-redux";
+import { useParams } from "react-router";
+
+import Card from "../../Components/Card/Index";
+import Dropdown from "./Dropdown.jsx";
+import { LibraryContext } from "./Context";
+
+import "./Cards.scss";
+
+function Cards(props) {
+  const params = useParams();
+
+  const { showUnmatched } = useContext(LibraryContext);
+
+  const [title, setTitle] = useState("");
+  const [cards, setCards] = useState([]);
+
+  /*
+    held in this state temp until animation
+    finishes swapping it.
+  */
+  const [newCards, setNewCards] = useState([]);
+
+  const [fetched, setFetched] = useState(false);
+  const [currentID, setCurrentID] = useState();
+  const [show, setShow] = useState(false);
+
+  const [throttleEventNewCardID, setThrottleEventNewCardID] = useState(false);
+
+  const { auth, ws } = useSelector(store => ({
+    auth: store.auth,
+    ws: store.ws
+  }));
+
+  const cardList = useRef(null);
+
+  const fetchCards = useCallback(async () => {
+    setNewCards([]);
+
+    try {
+      const config = {
+        headers: {
+          "authorization": auth.token
+        }
+      };
+
+      const res = await fetch(`/api/v1/library/${currentID}/media`, config);
+
+      if (res.status !== 200) {
```

Comment:
shouldnt we save the error if its not 200?

Reply:
Errors across the entire app aren't implemented properly right now, haven't come around to 'em yet. For now I simply just check if it was successful or not.